.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "prettier.requireConfig": true,
-  "coverage-gutters.showLineCoverage": true
+  "coverage-gutters.showLineCoverage": true,
+  "angular.enable-strict-mode-prompt": false
```

Comment:
I'm not against this - but I wonder if we should either have that be a user setting or at least try to remember to remove this once we get everything in strict mode.

Reply:
oops I accidentally committed this after blindly dismissing whatever question vscode asked me 😅